worker/include/RTC/TransportCongestionControlClient.hpp

```diff
@@ -49,7 +49,8 @@ namespace RTC
 		TransportCongestionControlClient(
 		  RTC::TransportCongestionControlClient::Listener* listener,
 		  RTC::BweType bweType,
-		  uint32_t initialAvailableBitrate);
+		  uint32_t initialAvailableBitrate,
+		  uint32_t maxAvailableBitrate);
```

Comment:
cosmetic, I would rename this argument to `maxOutgoingBitrate` for consistency with how the method is being called.

Reply:
:+1: